db/migrate/20201215122646_add_reputation_token_context_key.rb

```diff
@@ -0,0 +1,6 @@
+class AddReputationTokenContextKey < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_reputation_tokens, :context_key, :string, null: false
+    add_index :user_reputation_tokens, %i[context_key user_id], unique: true
+  end
+end
```

Comment:
You can add this to the main migration as that's in this PR too.

Reply:
Doh! Will update